### ISSUE TITLE
suggestion: give the relevant marker another color

### ISSUE THREAD

**Issue by meulie** [xbgmsharp/piwigo-openstreetmap#67]
example URL: http://pub.meulie.net/picture/89-01_082_op_den_ijsel_bij_zutphen

I have 2 images very close to each other, and some more in the direct neighbourhood. Currently all markers are 100% identical, which makes it hard to tell which marker belongs to the image on that page.
Is it an idea to give that one marker a different appearance?

**Comment by xbgmsharp** [comment]
Are you sing the latest github version?
I guess as this is bug introduce by a change for issue #64

**Comment by meulie** [comment]
Yup, I downloaded the latest version right before reporting this issue  :)

**Comment by xbgmsharp** [comment]
The latest version from where piwigo or github?

**Comment by xbgmsharp** [comment]
Please revert commit from #64, a quick fix would be to replace in file picture.inc.php on line 129:
```
$js_data = osm_get_items($page);
```
by
```
$js_data = array(array($lat, $lon, null, null, null, null, null, null));
```
https://github.com/xbgmsharp/piwigo-openstreetmap/blob/master/picture.inc.php#L129

**Comment by meulie** [comment]
I was referring to the latest version from github  :)

reverting #64 does remove all the other pins, and leaves only the pin of the image on the page.
